integration/test_collection_batch.py

```diff
@@ -39,33 +41,35 @@ def numpy(self) -> "MockNumpyTorch":
 
 
 @pytest.fixture(scope="function")
-def client_sync_indexing() -> Generator[weaviate.WeaviateClient, None, None]:
+def collection() -> Generator[weaviate.Collection, None, None]:
     client = weaviate.connect_to_local()
     client.collections.delete_all()
     client.collections.create(
-        name="Test",
+        name="CollectionBatchTest",
         properties=[
             Property(name="name", data_type=DataType.TEXT),
             Property(name="age", data_type=DataType.INT),
         ],
-        references=[ReferenceProperty(name="test", target_collection="Test")],
+        references=[ReferenceProperty(name="test", target_collection="CollectionBatchTest")],
     )
-    yield client
+    yield client.collections.get("CollectionBatchTest")
     client.collections.delete_all()
 
 
 @pytest.fixture(scope="function")
-def client_async_indexing() -> Generator[weaviate.WeaviateClient, None, None]:
-    client = weaviate.connect_to_local(port=8090, grpc_port=50060)
+def mt_collection() -> Generator[weaviate.Collection, None, None]:
+    client = weaviate.connect_to_local()
     client.collections.delete_all()
     client.collections.create(
-        name="BatchTestAsync",
-        vectorizer_config=Configure.Vectorizer.none(),
+        name="CollectionBatchTestMT",
         properties=[
-            Property(name="text", data_type=DataType.TEXT),
+            ReferenceProperty(name="test", target_collection="CollectionBatchTestMT"),
+            Property(name="name", data_type=DataType.TEXT),
+            Property(name="age", data_type=DataType.INT),
         ],
+        multi_tenancy_config=Configure.multi_tenancy(enabled=True),
```

Comment:
this could also be handled by one factory fixture

Reply:
If we have the chance, we should definitely revisit the test-refactoring work we tried a few months back